libvast/test/fixtures/events.cpp

```diff
@@ -87,6 +87,14 @@ record_type events::bro_conn_log_layout() {
   return const_bro_conn_log_slices[0]->layout();
 }
 
+slice_list events::copy(slice_list xs) {
+  slice_list result;
```

Comment:
I actually don't find that the type alias here reduces complexity. Now I have to look about what `slice_list` is...a `vector<table_slice_ptr>` or a `list<table_slice_ptr>`? Or maybe const pointer? It might feel dull to be repetitive, but I personally find it much clearer.